tests/foreman/ui/test_registration.py

```diff
@@ -0,0 +1,331 @@
+"""Tests for registration.
+
+:Requirement: Registration
+
+:CaseLevel: Acceptance
+
+:CaseComponent: Registration
+
+:CaseAutomation: Automated
+
+:CaseImportance: Critical
+
+:Team: Rocket
+
+:TestType: Functional
+
+:Upstream: No
+"""
+import datetime
```

Comment:
PRT failure `AttributeError: module 'datetime' has no attribute 'strptime'` looks related to this
```suggestion
from datetime import datetime
```

Reply:
Updated.